### DISCUSSION TITLE
My school allows importing certificates

### DISCUSSION THREAD

**Discussion by schoolexploitkid** [3kh0/ext-remover#1221] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
chrome://certificate-manager/clientcerts/platformclientcerts, it allows certificate importing, is there any way to exploit this? (for example, vpn, etc.)

**Comment by CGRmags** [comment]
shut up

On Mon, Oct 28, 2024 at 10:04 AM schoolexploitkid ***@***.***>
wrote:

> chrome://certificate-manager/clientcerts/platformclientcerts, it allows
> certificate importing, is there any way to exploit this? (for example, vpn,
> etc.)
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/3kh0/ext-remover/discussions/1221>, or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/BDXUJBAPP7AKH42J7NAKTH3Z5ZVCTAVCNFSM6AAAAABQX4WRFKVHI2DSMVQWIX3LMV43ERDJONRXK43TNFXW4OZXGM4DMNBVGQ>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***>
>

-- 
NOTICE: This email message and/or its attachments may contain information 
that is confidential or restricted. It is intended only for the individuals 
named as recipients in the message. If you are NOT an authorized recipient, 
you are prohibited from using, delivering, distributing, printing, copying, 
or disclosing the message or content to others and must delete the message 
from your computer. If you have received this message in error, please 
notify the sender by return email.

**Comment by HelloIloveGimkit2** [comment, marked as answer]
here's an answer to why bro said shut up, it's cuz it's just importing a certificate which is for allowing websites to load without that annoying "This website uses unsafe connection" but the problem with that is that iboss has a Blocklist so even if you load the website it still blocks the site so you can't just bypass it with a certificate

Idiot terms: Iboss has Blocklist so even if add certificate it will redirect to Iboss block website

**Reply by schoolexploitkid** [reply]
hm what abt protonvpn

**Comment by akabutnicer** [comment]
> chrome://certificate-manager/clientcerts/platformclientcert

Yeah, just wait.
